base/applications/cmdutils/tasklist/tasklist.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#define WIN32_NO_STATUS
+#include <windows.h>
+#include <ntndk.h>
+
+#include <conutils.h>
+
+#include "resource.h"
+
+#define NT_SYSTEM_QUERY_MAX_RETRY 5
+
+#define COLUMNWIDTH_IMAGENAME 25
+#define COLUMNWIDTH_PID 8
+#define COLUMNWIDTH_SESSION 11
+#define COLUMNWIDTH_MEMUSAGE 12
+
+#define RES_STR_MAXLEN 64
+
+static WCHAR opHelp[] = L"?";
+static WCHAR opVerbose[] = L"v";
+static PWCHAR opList[] = {opHelp, opVerbose};
```

Comment:
Since you are not also using the "opXXX" variables individually explicitly in the rest of the code, maybe just consider doing something like:
```suggestion
static PCWSTR opList[] = {L"?", L"v"};
```